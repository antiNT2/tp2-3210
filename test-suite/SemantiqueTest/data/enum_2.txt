enum A {
    B,
    C,
    D
}
B var;