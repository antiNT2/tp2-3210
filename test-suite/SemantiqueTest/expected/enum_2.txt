Identifier var has been declared with the type B that does not exist